tests/unit/utils/test_parse_nftoken_id.py

```diff
@@ -0,0 +1,27 @@
+"""Test the parse_nftoken_id util."""
+from __future__ import annotations
+
+from unittest import TestCase
+
+from xrpl.utils.parse_nftoken_id import parse_nftoken_id
+
+
+class TestParseNFTokenID(TestCase):
+    """Test parse_nftoken_id."""
+
+    def test_parse_nftoken_id_successful(self: TestParseNFTokenID) -> None:
+        nft_id = "000B0539C35B55AA096BA6D87A6E6C965A6534150DC56E5E12C5D09E0000000C"  # noqa:E501
```

Comment:
This shouldn't be triggering E501 - it's only 84 characters long and the limit is set to 88.